libcloud/test/compute/test_openstack.py

```diff
@@ -2755,6 +2770,35 @@ def _v1_1_slug_servers_EX_DISK_CONFIG(self, method, url, body, headers):
                 httplib.responses[httplib.OK],
             )
 
+    def _v1_1_slug_servers_EX_FILES(self, method, url, body, headers):
+        if method == "POST":
+            body = u(body)
+            personality = (
+                '"personality": ['
```

Comment:
Probably a bit more robust way would be to json.dumps the expected body and json.loads the response and compare the parsed dicts (otherwise the assertion may break in case we change the json library or similar).

Reply:
You are right.
I fixed the test.